packages/mobile/src/transactions/TransactionsList.tsx

```diff
@@ -254,21 +254,17 @@ export class TransactionsList extends React.PureComponent<Props> {
       const queryDataTxHashes = new Set(transactions.map((tx) => tx.hash))
       const standbyTxs = standbyTransactions
         .filter((tx) => {
-          const isForQueriedCurrency =
```

Comment:
Does anyone have context on why this was here? It looks like `blockchain-api` already does this filtering, so it's redundant, but I might be missing something. I didn't notice anything wrong after removing it but would love input from someone more familiar with this!

Reply:
Yes this is needed for standby transactions (confirming...) which are local to the device, otherwise confirming transactions show up in both the home and CELO feed.
And now this filtering should only apply to the CELO feed.